### ISSUE TITLE
Provide a clue to remind user if they want to delete the original one once the clone is complete

### ISSUE THREAD

**Issue by v-xianya** [microsoft/AzureStorageExplorer#2442]
**Storage Explorer Version:** 1.11.2
**Build:** [20191215.1](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3323607)
**Branch:** hotfix/1.11.2
**Platform/OS:** Windows 10/ Linux Ubuntu 18.04/macOS High Sierra
**Architecture:** ia32/x64
**Regression From:** Not a regression

**Steps to reproduce:**
1. Expand one storage account -> File Shares.
2. Create one file share -> Clone the file share with new name.
 
**Expect Experience:**
Provide a clue to remind user if they want to delete the original one after the clone is complete.

**Actual Experience:**
No any clue provided.

**Comment by MRayermannMSFT** [comment]
Per discussion with Jing, won't fix.
